content/blog/2018/06/2018-06-26-new-api-token-system.adoc

```diff
@@ -0,0 +1,86 @@
+---
+:layout: post
+:title: "Security Hardening: New API token system in Jenkins 2.129+"
+:tags:
+- community
+- core
+- developer
+- security
+- upgrade
+:author: wadeck
+---
+
+### Context
+
+There were two problems with the existing API token system. First, 
```

Comment:
Maybe a sentence about what the API token system is used for would be good before going straight into the problems.